web_widget_domain_editor_dialog/static/src/js/basic_fields.js

```diff
@@ -16,6 +16,9 @@ odoo.define("web_widget_domain_editor_dialog.basic_fields", function(require) {
             if (this.mode === "readonly") {
                 return this._super.apply(this, arguments);
             }
+            if (this.value == false) {
```

Comment:
If it's `undefined` it won't be `false` wich is a very specific property, so maybe it would be more appropiate something like this:
```suggestion
            if (!this.value) {
```

Reply:
Fixed. Thanks @chienandalu !